src/public/index.ts

```diff
@@ -47,11 +52,31 @@ function main() {
   // first screen event handler
   const fileInput = document.getElementById("importFile") as HTMLInputElement;
   const readButton = document.getElementById("readButton") as HTMLElement;
+  const showExampleButton = document.getElementById("showExampleButton") as HTMLElement;
   fileInput.addEventListener("change", uploader.showFile.bind(uploader));
   readButton.addEventListener("click", () => {
     uploader.readFile();
     entityPicker.showCard();
   });
+  showExampleButton.addEventListener("click", () => {
+    uploader.hideCard();
+    csvFileStructureExample.showCard();
+  });
+
+  // CSV example screen event handler
+  const showNodeExampleButton = document.getElementById("showNodeExampleButton") as HTMLElement;
+  const showEdgeExampleButton = document.getElementById("showEdgeExampleButton") as HTMLElement;
+  const exitExampleButton = document.getElementById("exitExampleButton") as HTMLElement;
+  showNodeExampleButton.addEventListener("click", () => {
+    csvFileStructureExample.showImageExample(EntityType.NODE);
+  });
+  showEdgeExampleButton.addEventListener("click", () => {
+    csvFileStructureExample.showImageExample(EntityType.EDGE);
+  });
+  exitExampleButton.addEventListener("click", () => {
+    csvFileStructureExample.hideCard();
+    uploader.showCard();
+  });
```

Comment:
Are you sure you don't need to remove the event listeners at some point? This is a BIG source of memory leaks.

Reply:
I ll check when we can remove

Reply:
little bit complicated as we are using anonymous function , to be able to remove listener need to create a function for each callback. 
I think the as the plugin is small and the number of eventLister is limited we can allow it.

What do you think ?

Reply:
Yes. It's a one time use and the user will not keep this page open for ages.